examples/A2C/train.py

```diff
@@ -14,22 +14,18 @@
 
 import time
 from learner import Learner
+import parl
 
 
 def main(config):
     learner = Learner(config)
     assert config['log_metrics_interval_s'] > 0
 
-    try:
-        while not learner.should_stop():
-            start = time.time()
-            while time.time() - start < config['log_metrics_interval_s']:
-                learner.step()
-            learner.log_metrics()
-        learner.close()
-
-    except KeyboardInterrupt:
-        learner.close()
+    while not learner.should_stop():
+        start = time.time()
+        while time.time() - start < config['log_metrics_interval_s']:
+            learner.step()
+        learner.log_metrics()
```

Comment:
why we remove learner.close()? It will cause that csv logger won't flush data to file.

Reply:
We have removed CSV logger in the code, as we are using tensorboard for visualizing the training status.

Reply:
ok